framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2997,22 +2999,24 @@ VkResult VulkanReplayConsumerBase::OverrideWaitForFences(PFN_vkWaitForFences
         modified_fences      = valid_fences.data();
     }
 
-    if (original_result == VK_SUCCESS)
-    {
-        // Ensure that wait for fences waits until the fences have been signaled (or error occurs) by changing the
-        // timeout to UINT64_MAX.
```

Comment:
Our grandchildren would still have been waiting :-O.

Reply:
Still appears to me a better outcome than trying to continue with a broken synchronization after just one second, as this can result in silently giving bad data. Also, one second might seem like an eternity to developers working on highend GPUs in silicon, but eg for those of us who work on simulators where a single frame can take days to process, it is not. Please reconsider.

Reply:
I'm with Per here that if capture eventually succeeded, we should ensure here that replay eventually succeeds and that's what this original code ensured, if I understand it correctly.